stork/src/base/StorkApp.C.app

```diff
@@ -14,21 +14,11 @@ StorkApp::validParams()
 
 StorkApp::StorkApp(InputParameters parameters) : MooseApp(parameters)
 {
-  StorkApp::registerAll(_factory, _action_factory, _syntax);
+  ModulesApp::registerAllObjects<StorkApp>(_factory, _action_factory, _syntax);
 }
 
 StorkApp::~StorkApp() {}
 
-void
-StorkApp::registerAll(Factory & f, ActionFactory & af, Syntax & syntax)
-{
-  ModulesApp::registerAll(f, af, syntax);
-  Registry::registerObjectsTo(f, {"StorkApp"});
-  Registry::registerActionsTo(af, {"StorkApp"});
-
-  /* register custom execute flags, action syntax, etc. here */
-}
-
```

Comment:
This was not the right fix. `ModulesApp::registerAll` should have been modified directly. Stork is now broken for new users.

Reply:
@Naktakala should have a PR up fixing this shortly.

Reply:
This change was made based on the existing deprecation message for `StorkApp::registerAll`.